jax/_src/numpy/lax_numpy.py

```diff
@@ -3782,8 +3782,32 @@ def array_equal(a1: ArrayLike, a2: ArrayLike, equal_nan: bool = False) -> Array:
   return reductions.all(eq)
 
 
-@util.implements(np.array_equiv)
 def array_equiv(a1: ArrayLike, a2: ArrayLike) -> Array:
+  """Check if two arrays can be broadcasted to the same shape and element-wise equal.
```

Comment:
I feel like this text is still a little ambiguous. What about if we simplify the top text and add a clarification below, as follows:

```suggestion
  """Check if two arrays are element-wise equal.
  
  This function will return ``False`` if the input arrays cannot be broadcasted
  to the same shape.
```